client/src/containers/mealList.js

```diff
@@ -3,37 +3,48 @@ import { connect } from 'react-redux';
 import { fetchMeals } from '../actions/index';
 import { Link } from 'react-router-dom';
 import _ from 'lodash';
+import {GridList, GridTile} from 'material-ui/GridList';
+import IconButton from 'material-ui/IconButton';
+import Subheader from 'material-ui/Subheader';
+import StarBorder from 'material-ui/svg-icons/toggle/star-border';
+
+const styles = {
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'space-around',
+  },
+  gridList: {
+    width: 500,
+    height: 450,
+    overflowY: 'auto',
+  },
+};
 
 class MealList extends Component {
   componentDidMount() {
     this.props.fetchMeals();
   }
 
-  renderMeals() {
-    return _.map(this.props.meals, meal => {
-      return (
-        <li key={meal.id}>
-          <Link to={`/meals/${meal.id}`}>
-            {meal.name}
-          </Link>
-          <div>
-            <img src={meal.images} width="200" height="200" />
-          </div>
-          <div>
-            {meal.description}
-          </div>
-        </li>
-      );
-    });
-  }
-
   render() {
     return (
-      <div>
-        <h1>List of Food</h1>
-        <ul>
-          {this.renderMeals()}
-        </ul>
+      <div style={styles.root}>
+        <GridList
+          cellHeight={180}
+          style={styles.gridList}
+        >
+          <Subheader>List of Meals</Subheader>
+          {_.map(this.props.meals, (meal) => (
+            <GridTile
+              key={meal.name}
+              title={<Link to={`/meals/${meal.id}`} style={{color:"white", textDecoration: "none"}}>{meal.name}</Link>}
+              subtitle={<span>by <b>JORV</b></span>}
+              actionIcon={<IconButton><StarBorder color="white" /></IconButton>}
```

Comment:
part of material UI, i can refactor the entire thing as a method and call it in the render function  for an easier read but i dont think i can change the props